src/scene/shader-lib/chunks/standard/frag/litShaderArgs.js

```diff
@@ -0,0 +1,49 @@
+export default /* glsl */`
+
+struct IridescenceArgs
```

Comment:
add comments to this file please, for each member.

Reply:
Done